bgpd/bgp_fsm.c

```diff
@@ -1376,6 +1377,14 @@ int bgp_start(struct peer *peer)
 		return 0;
 	}
 
+	if (peer->bgp &&
+	    peer->bgp->vrf_id == VRF_UNKNOWN) {
+		zlog_err(
```

Comment:
Won't this result in a million zlog_err's being displayed if the vrf is not cnfigured?  This should be debug guarded

Reply:
ok